src/Plugin/EntityBrowser/Widget/View.php

```diff
@@ -209,4 +210,41 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
     $this->selectEntities($entities, $form_state);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $options = [];
+    // Get only those enabled Views that have entity_browser displays.
+    $displays = Views::getApplicableViews('entity_browser_display');
+    foreach ($displays as $display) {
+      list($view_id, $display_id) = $display;
+      $view = $this->entityManager->getStorage('view')->load($view_id);
+      $options[$view_id . '.' . $display_id] = $this->t('@view : @display', array('@view' => $view->label(), '@display' => $view->get('display')[$display_id]['display_title']));
+    }
```

Comment:
> In other words: is there any real use case for Entity Browser without Views?

There are many. Views should not be a hard dependency of Entity browser. We should check if Views are enabled and warn users if they're trying to use views widget without it.